public/index.html

```diff
@@ -1,22 +1,25 @@
+      ;NREUM.loader_config={accountID:"<%= htmlWebpackPlugin.options.newRelic.accountId %>",trustKey:"<%= htmlWebpackPlugin.options.newRelic.trustKey %>",agentID:"<%= htmlWebpackPlugin.options.newRelic.agentId %>",licenseKey:"<%= htmlWebpackPlugin.options.newRelic.licenseKey %>",applicationID:"<%= htmlWebpackPlugin.options.newRelic.applicationId %>"}
```

Comment:
Should the options field be there? In our webpack config, aren't we putting these fields in htmlWebpackPlugin.newRelic?

Reply:
htmlWebpackPlugin.options hash contains all of the variables passed to the plugin
https://github.com/jantimon/html-webpack-plugin#writing-your-own-templates